packages/central-server/app/sync/CentralSyncManager.js

```diff
@@ -359,6 +365,8 @@ export class CentralSyncManager {
 
       await this.waitForPendingEdits(tick);
 
+      const lookupTickRange = await getLookupSourceTickRange(this.store, since, tick)
```

Comment:
Real nit but because there's just two things here, I kinda like getting it in full rather than having the opaque spread below
```suggestion
      const { minSourceTick, maxSourceTick } = await getLookupSourceTickRange(this.store, since, tick)
```

_or_ (and definitely equally happy with this) having it stored under the key `lookupTickRange`. Just don't love the spread of what is an invisible set of properties below.

Reply:
Yeah true it was to get around that inline await due to having to default it sumway or another might be a cleaner way tho

Reply:
I could return {} early in the getLookupSourceTickRange thats the other option - but wans't sure